demo/server/DB.re

```diff
@@ -71,6 +71,63 @@ let findOne = (notes, id) => {
     Lwt_result.fail("Note with id " ++ Int.to_string(id) ++ " not found")
   };
 };
+let addNote = (~title, ~content) => {
+  let%lwt notes = readNotes();
+  let notes =
+    Result.map(
+      notes => {
+        let length = List.length(notes);
+        let note: Note.t = {
+          id: length,
+          title,
+          content,
+          updated_at: Unix.time(),
+        };
+        [note, ...notes];
+      },
+      notes,
+    );
+  Cache.set(notes);
+  Lwt_result.lift(notes |> Result.map(notes => notes |> List.hd));
+};
```

Comment:
This doesn't addNote into the DB, but rather into the Cache. Which is perfectly fine, but no need to do `Lwt_result.lift(notes |> Result.map(notes => notes |> List.hd));` right?